flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkManifestUtil.java

```diff
@@ -62,9 +62,10 @@ static ManifestFile writeDataFiles(OutputFile outputFile, PartitionSpec spec, Li
   }
 
   static ManifestOutputFileFactory createOutputFileFactory(Table table, String flinkJobId, int subTaskId,
-                                                           long attemptNumber) {
+                                                           String operatorUniqueId, long attemptNumber) {
     TableOperations ops = ((HasTableOperations) table).operations();
-    return new ManifestOutputFileFactory(ops, table.io(), table.properties(), flinkJobId, subTaskId, attemptNumber);
+    return new ManifestOutputFileFactory(ops, table.io(), table.properties(), flinkJobId, subTaskId,
+        operatorUniqueId, attemptNumber);
```

Comment:
Nit: Can we fit all of the arguments on one line if they're placed on the next line?

```java
    return new ManifestOutputFileFactory(
        ops, table.io(), table.properties(), flinkJobId, subTaskId, operatorUniqueId, attemptNumber);
```

You could also make the new parameter into `operatorUid` (or `operatorId`) to shorten it.

Not a blocker by any means but would be nice if possible. =)

Reply:
@kbendick Thanks, I'll fix it